test/framework/src/main/java/org/elasticsearch/transport/nio/channel/NioSocketChannel.java

```diff
@@ -47,6 +49,22 @@ public NioSocketChannel(String profile, SocketChannel socketChannel, SocketSelec
     }
 
     @Override
+    public void sendMessage(BytesReference reference, ActionListener<TcpChannel> listener) {
+        // TODO: temporar
```

Comment:
can we have a more verbose TODO?

Reply:
Yeah. I'm not sure what happened to the rest of the comment.